app/models/booking_field.rb

```diff
@@ -68,6 +68,13 @@ class BookingField < ApplicationRecord
                            where(field_id:, date:).where.not(id: current_id)
                          }
 
+  scope :yet_booking, lambda {|field_id|
+    if field_id.present?
+      where(field_id:)
+        .where(status: :approval)
+    end
```

Comment:
```suggestion
    return if field_id.blank?
      where(field_id:)
        .where(status: :approval)
```
Viết scope cho `where(status: :approval)`